Jenkinsfile

```diff
@@ -84,3 +83,30 @@ demos['databound'] = {
 }
 
 parallel(demos)
+
+node('docker') {
```

Comment:
I am not sure you want to run publishing for pull requests except this one

Reply:
Agree.

Reply:
See my next commit and tell me if it's a fine for you